cli/src/cli.rs

```diff
@@ -0,0 +1,34 @@
+use clap::Parser;
+use color_eyre::{eyre::eyre, Report, Result};
+use std::path::PathBuf;
+
+use crate::config::Config;
+
+/// Bob configuration
+#[derive(Debug, Parser, Clone)]
+#[command(author = "Romanov Simeon <ArchArcheoss@proton.me>")]
+#[command(version, about, long_about)]
```

Comment:
I suggest to extend `version`. It is common problem to detect, which version is used during the debug. I suggest to add information about build time, commit hash, and branch/tag name